app/models/avatar.rb

```diff
@@ -0,0 +1,11 @@
+# A model for storing user avatars in the database
+class Avatar < ApplicationRecord
+  VALID_MIME_TYPES = %w[image/png image/jpg image/jpeg image/gif].freeze
+  MAX_FILE_SIZE_MEGABYTES = 5
+
+  validates :filename, :filesize, :file, :mime_type, presence: true
```

Comment:
```suggestion
  validates :filename, :file_size, :file, :mime_type, presence: true
```
And everywhere else

Reply:
Should I do the same for the `filename`?

Reply:
No, filename is an English word https://www.oxfordlearnersdictionaries.com/definition/english/filename?q=filename

Reply:
Done